Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +36,48 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message))
+            {
+                return "";
+            }
+
+            string[] splitMessage = message.Split(new string[] { System.Environment.NewLine }, System.StringSplitOptions.None);
+            StringBuilder sb = new StringBuilder();
+            int longStringIndex = LongestStringIndex(splitMessage);
+            
+            sb.Append("+").Append('-', splitMessage[longStringIndex].Length + padding*2).AppendLine("+");
+            for(int i = 0; i < padding; i++)
+            {
+                sb.Append("|").Append(' ', splitMessage[longStringIndex].Length + padding*2).AppendLine("|");
+            }
+            for(int i = 0; i < splitMessage.Length; i++)
+            {
+                sb.Append("|").Append(' ', padding).Append(splitMessage[i]);
+                sb.Append(' ', splitMessage[longStringIndex].Length - splitMessage[i].Length);
+                sb.Append(' ', padding).AppendLine("|");
+            }
+            for (int i = 0; i < padding; i++)
+            {
+                sb.Append("|").Append(' ', splitMessage[longStringIndex].Length + padding*2).AppendLine("|");
+            }
+            sb.Append("+").Append('-', splitMessage[longStringIndex].Length + padding*2).AppendLine("+");
+
+            return sb.ToString();
+        }
+        public static int LongestStringIndex(string[] message)
+        {
+            if (message.Length == 1) return 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5156603)